modules/server/src/graphqlRoutes.js

```diff
@@ -276,15 +276,21 @@ export const createSchemasFromConfigs = async ({
 		 * Federated Network Search
 		 */
 		if (enableNetworkAggregation) {
+			const networkConfig = configsFromFiles[ConfigProperties.NETWORK_AGGREGATION];
```

Comment:
think this may help reading
```suggestion
			const networkConfigsObj = configsFromFiles[ConfigProperties.NETWORK_AGGREGATION];
```